style.css

```diff
@@ -62,9 +466,260 @@
   color: var(--colorDeepBrown);
   font-family: var(--fontFamilyLato);
   font-size: 20px;
-  margin: 0 0 0 0;
+  margin: 0;
 }
 
 .white-overlay {
+  height: 650px;
   background: rgba(255, 255, 255, 0.9);
 }
+
+@media screen and (min-width: 768px) {
+  .about-info {
+    background-color: white;
+    border: 2px solid var(--colorGrey);
+    width: 50%;
+  }
+
+  .desktop-header {
+    display: block;
+  }
+
+  .desktop-secondary-nav {
+    background-color: white;
+    height: 90px;
+    position: fixed;
+    width: 100%;
+    top: 40px;
+    -webkit-box-shadow: 0 19px 20px -9px rgba(0, 0, 0, 0.32);
+    box-shadow: 0 19px 20px -9px rgba(0, 0, 0, 0.32);
+    z-index: 1;
+  }
+
+  .desktop-top-nav {
+    background-color: var(--colorDeepBrown);
+    height: 40px;
+    position: fixed;
+    width: 100%;
+    top: 0;
+    z-index: 1;
+  }
+
+  .home-grey-background {
+    display: none;
+  }
+
+  .desktop-home-background {
+    background-image: url("images/home-background.jpg");
+    background-repeat: repeat;
+    height: 100vh;
+    width: 100%;
+  }
+
+  .desktop-overlay {
+    height: 100vh;
+    background: rgba(255, 255, 255, 0.9);
+  }
+
+  .feature-speaker-list-parent {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: repeat(3, 1fr);
+    grid-column-gap: 46px;
+    grid-row-gap: 19px;
+    list-style: none;
+    padding: 0;
+    margin-left: 20%;
+    margin-right: 20%;
+  }
+
+  .footer-content {
+    display: flex;
+    justify-content: space-around;
+    margin-top: 50px;
+    margin-bottom: 50px;
+  }
```

Comment:
- [ ] Kindly change the background color for the `footer` section to a darker one on the about page, to match the original design. Notes that in the About Page: It has a dark background and in the Home Page: It has a white background.

Your design
![image](https://user-images.githubusercontent.com/35535130/158814670-c1aba4d7-543a-4095-80e5-49c3ae548faf.png)


Original design
![image](https://user-images.githubusercontent.com/35535130/158814599-77aff4a5-25bd-4e6e-bb16-b1169f979305.png)